developers/weaviate/current/configuration/authentication.md

```diff
@@ -134,17 +123,55 @@ services:
       AUTHENTICATION_OIDC_SKIP_CLIENT_ID_CHECK: 'false'
 ```
 
+#### Weaviate OpenID endpoint
+
+If you have authentication enabled, you can request all details needed from the following endpoint:
+
+```bash
+$ curl http://localhost:8080/v1/.well-known/openid-configuration
+```
+
 ### How to use
 
-1. Obtain a valid token from the token issuer you configured.
-2. Send this token along any REST request in the Header like so: `Authorization: Bearer <token>`. Make sure to replace `<token>` with your actual token.
+OIDC authentication requires obtaining a valid token from the token issuer so that it can be sent in the header of any REST request.
+
+The OIDC standard allows for many different methods *(flows)* of obtaining a token. The appropriate method can vary depending on your situation, including configurations at the token issuer, and your requirements. 
+
+While it is outside the scope of our documentation to cover every OIDC authentication flow, some possible options are to:
+- Use the `resource owner password flow` for trusted applications. 
+- Or use the `hybrid flow` if Microsoft Azure AD is your token issuer or if you would like to prevent exposing passwords.
+
+We outline the steps below for both methods of obtaining a token.
```

Comment:
Can we link the python script here (or if it is easy, a webinterface)? Might be easier for some users

Reply:
Done :)